packages/compiler/tsconfig.json

```diff
@@ -6,5 +6,6 @@
     "tsBuildInfoFile": "temp/tsconfig.tsbuildinfo",
     "types": ["node", "mocha"]
   },
-  "include": ["src/**/*.ts", "test/**/*.ts"]
+  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "exclude": ["src/init/templates/*/*"]
```

Comment:
Todo do we want to move those templates to a dedicated package maybe?

Reply:
I think that would be cleaner specially if we want to add some e2e test to make sure the templates generate correclty.